app/controllers/analytics/facilities_controller.rb

```diff
@@ -83,23 +83,13 @@ def set_facility
   end
 
   def set_cohort_analytics(period, prev_periods)
-    @cohort_analytics =
-      set_analytics_cache(analytics_cache_key_cohort(period)) {
-        @facility.cohort_analytics(period, prev_periods)
-      }
+    @cohort_analytics = @facility.cohort_analytics(period, prev_periods)
   end
 
   def set_dashboard_analytics(period, prev_periods)
-    @dashboard_analytics =
-      set_analytics_cache(analytics_cache_key_dashboard(period)) {
-        @facility.dashboard_analytics(period: period,
-                                      prev_periods: prev_periods,
-                                      include_current_period: @show_current_period)
-      }
-  end
-
-  def analytics_cache_key
-    "analytics/facilities/#{@facility.id}"
+    @dashboard_analytics = @facility.dashboard_analytics(period: period,
+                                                         prev_periods: prev_periods,
+                                                         include_current_period: @show_current_period)
```

Comment:
I think that could make sense, but it is a decent amount of work to make sure we keep all the date logic consistent to avoid fence-post errors and the like.  I also think we can consolidate some of these queries down once we get their interfaces more consistent, which would make the Period refactoring easier.

Added a story here: https://app.clubhouse.io/simpledotorg/story/1183/refactor-older-reporting-queries-to-use-a-range-of-periods